pkg/ingress2httpproxy/ingress2httpproxy_test.go

```diff
@@ -0,0 +1,52 @@
+package ingress2httpproxy
```

Comment:
i would consider adding tests for things like:

* if a wildcard dns is passed in, does that get used?
* domain parsing works for various formats / hits any validations

etc. really just some tests where logic that may extend outside one to one mappings occur to ensure cases are being handled

Reply:
added the additional test cases